pytest_answer.ini

```diff
@@ -0,0 +1,10 @@
+[pytest]
+local-dir=/home/travis/build/yt-project/yt/answer-store
+addopts =
+    -s
+    -v
+    --with-answer-testing
+    -rsfE
```

Comment:
isn't the `-s` part redundant ?
I'd suggest commenting each flag and/or having a link to the official docs (I'd prefer comments as they don't require to be online)

Reply:
this file is now gone right ? In which case, you can resolve this @jcoughlin11